css/main.css

```diff
@@ -2209,23 +2209,38 @@ li.tg-rss a{background:#ff6600;}
 	padding:15px;
 	list-style-type:none;
 }
+
+/* For medium devices and up */
 @media (min-width: 991px){
   .gold{
       margin-left: 12%;
       width: 50% !important;
   }
+  .pipal{
+    margin-left: 37%;
+  }
 }
+
+/* For small devices */
 @media (max-width: 991px){
   .gold {
       margin-left: 25%;
       width: 50% !important;
   }
+  .pipal {
+    margin-left: 35%;
+  }
 }
+
+/* For extra small device like mobiles */
 @media (max-width: 699px){
   .gold {
       margin-left: 0%;
       width: 100% !important;
   }
+  .pipal{
+    margin-left: 25%;
```

Comment:
You need to try making `margin-left: 0%` here as `25%` does not look good on mobile devices.
![image](https://user-images.githubusercontent.com/10004530/31589228-d2bf26cc-b21b-11e7-9f8e-68631bd2d783.png)